src/sql/azdata.proposed.d.ts

```diff
@@ -725,6 +725,10 @@ declare module 'azdata' {
 			 * Extension can store additional information that the provider needs to uniquely identify a table.
 			 */
 			[key: string]: any;
+			/**
+			 * Table Node subtype (Temporal/GraphNode/GraphEdge)
```

Comment:
Remove the example types, that's not really necessary and could easily get out of date if they're changed or more are added